_topic_map.yml

```diff
@@ -331,7 +331,38 @@ Topics:
 - Name: Disabling the web console
   File: disabling-web-console
   Distros: openshift-enterprise,openshift-webscale,openshift-origin
-
+---
+Name: Security
+Dir: security
+Distros: openshift-enterprise,openshift-webscale,openshift-origin,openshift-dedicated
+Topics:
+- Name: Container security
+  Dir: container_security
+  Topics:
+  - Name: Understanding container security
+    File: security-understanding
+  - Name: Container hosts and VMs
+    File: security-hosts-vms
+  - Name: Hardening RHCOS
```

Comment:
My guidance later about using an attribute for RHCOS does _not_ apply to this file. Attributes are not recognized in the _topic_map.yml file.

However, if this file should appear both in enterprise, etc. and origin (once you confirm the distros), then we'll need to get a little fancy. I think it should work to list the file twice, with different headings and different distros:

```
  - Name: Hardening Red Hat Enterprise Linux CoreOS
    File: security-hardening
    Distros: openshift-enterprise,openshift-webscale,openshift-dedicated
  - Name: Hardening Fedora CoreOS
    File: security-hardening
    Distros: openshift-origin
```
And note that I think it should be written out here, not just RHCOS.

I know this is complicated, so let me know if you have any questions on this.

Reply:
Sure. That makes sense. You need separate entries since you can't just do it with an attribute.

Reply:
I'll expand out the full "Red Hat Enterprise Linux CoreOS" in the Name and wait until I hear from Vikram about the Distros.

Reply:
Done.